client/header/activity-panel/index.js

```diff
@@ -2,17 +2,16 @@
  * External dependencies
```

Comment:
It's probably a good idea to add some smoke test instructions for this after such a big refactor

Reply:
@becdetat There are testing instructions in the PR and added to `TESTING-INSTRUCTIONS.md`.

I do think that it's worth a good general smoke testing of the app.  Are you suggesting copying over all the generic smoke testing instructions here from the p2 thread or is there something you'd like more specific testing instructions on?

Reply:
I meant testing the activity panel - it doesn't seem to be covered by the test instructions